gitstatus.py

```diff
@@ -10,9 +10,7 @@
 gitsym = Popen(['git', 'symbolic-ref', 'HEAD'], stdout=PIPE, stderr=PIPE)
```

Comment:
There is most quick way to get info are we in git repo or not
```
isgit = Popen(['git', 'name-rev'], stdout=PIPE, stderr=PIPE)
_, _ = isgit.communicate()                                  
if isgit.returncode != 0:                                   
    sys.exit(0)                                             
```
IMO it should be right after `import sys`